tests/routes/search_test.py

```diff
@@ -90,6 +90,42 @@ def test_empty_search_path(self):
             }) \
             .post({}, headers=self.auth)
 
+    def test_sort_no_metadata(self):
+        # Adding metadata tag THINGS to test artifact to ensure it comes first
+        # on DESC search.
+        self.route_tester \
```

Comment:
Also this after I saw I missed a test case 8e1fddf9cf095e2459918bacbec86f9caadf2cf0